apps/arweave/src/ar_http_iface_client.erl

```diff
@@ -315,31 +322,34 @@ get_encrypted_full_block(Peer, Hash) when is_binary(Hash) ->
 		)
 	).
 
-get_txs(Peers, MempoolTXs, B) ->
+get_txs(_Peers, _MempoolTXs, _B, ?NO_TX) -> {ok, []};
```

Comment:
This would override the list of txids in the block header with an empty list.

Reply:
Ok. So a caller specifying `?NO_TX` will mean retain TXIDs, and if acquired block already has full TXs, retain & return them as well, but just don't fetch TXs in the current call. I'll update.

Reply:
Correct, except for the acquired block is not supposed to ever have full transactions here.